libraries/Microsoft.Bot.Connector/Authentication/SkillValidation.cs

```diff
@@ -144,6 +144,14 @@ public static async Task<ClaimsIdentity> AuthenticateChannelToken(string authHea
                 GovernmentAuthenticationConstants.ToBotFromEmulatorOpenIdMetadataUrl : 
                 AuthenticationConstants.ToBotFromEmulatorOpenIdMetadataUrl;
 
+            // Add allowed token issuers from configuration (if present)
+            if (authConfig.ValidTokenIssuers != null && authConfig.ValidTokenIssuers.Any())
+            {
+                var validIssuers = _tokenValidationParameters.ValidIssuers.ToList();
+                validIssuers.AddRange(authConfig.ValidTokenIssuers);
+                _tokenValidationParameters.ValidIssuers = validIssuers;
```

Comment:
This code to update "_tokenValidationParameters" is not thread safe. Consider making _tokenValidationParameters Lazy since Lazy type initializers are thread safe by default.

Reply:
Made "_tokenValidationParameters" to be a local variable since this is the only place where that field is being accessed.